crates/next-core/src/app_source.rs

```diff
@@ -112,7 +114,7 @@ async fn next_client_transition(
 
     Ok(NextClientTransition {
         is_app: true,
-        server_root,
+        server_root: server_path,
```

Comment:
@sokra I'm not sure this is correct. Perhaps this needs both `server_root` and `server_path` to be passed down?